templates/logback.xml.erb

```diff
@@ -21,5 +21,7 @@
   <root level="<%= @log_level %>">
     <appender-ref ref="FILE" />
   </root>
-
+  <% @log_levels.each do |key, level| -%>
+  <logger name="<%= key %>" level="<%= level %>"/>
+  <% end -%>
```

Comment:
Small nitpick: can we put this above the root logger in the file? Generally the root logger is the last thing configured in the logger config because it (figuratively speaking) catches the log messages that haven't been configured elsewhere.